cmd/agent/app/integrations.go

```diff
@@ -247,75 +228,24 @@ func pip(args []string, local bool) error {
 		color.NoColor = true
 	}
 
-	err := common.SetupConfig(confFilePath)
-	if err != nil {
-		fmt.Printf("Cannot setup config, exiting: %v\n", err)
-		return err
-	}
-
-	pipPath, err := getCommandPython()
+	pythonPath, err := getCommandPython()
 	if err != nil {
 		return err
 	}
-	tufPath, err := getTUFConfigFilePath()
-	if err != nil && !withoutTuf && !local {
-		return err
-	}
 
 	cmd := args[0]
 	implicitFlags := args[1:]
 	implicitFlags = append(implicitFlags, "--disable-pip-version-check")
 	args = append([]string{"-mpip"}, cmd)
 
-	if verbose {
-		args = append(args, "-vvv")
+	for i := 0; i < verbose; i++ {
+		args = append(args, "-v")
```

Comment:
Hmm, not sure this does the right thing. Might be better to do something like `'-' + ('v'*verbose)` in Python?